styles/components/input.css

```diff
@@ -0,0 +1,14 @@
+:root{
+    --primary-color : #7C3AED;
+    --secondary-color : #F5F3FF;
+}
+
+.field{
+    border:1px solid #d3d3d3;
+    padding:0.5rem;
+    margin:0.5rem;
+    font-family: 'Montserrat', sans-serif;   
```

Comment:
Use `inherit` to inherit the custom font specified for the body instead of specifying it explicitly every time.